components/image/picker/src/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import ImagePlaceholder from '@schibstedspain/sui-image-placeholder'
+
+class ImagePicker extends Component {
+  constructor (props) {
+    super(props)
+
+    const { images } = props
+    let filteredImages = []
+    let selected = -1
+
+    if (images.length > 0) {
+      filteredImages = images.filter(image => image.src && image.thumb)
+
+      if (filteredImages.length > 0) {
+        selected = filteredImages.findIndex(({ selected }) => selected)
+        selected = (selected === -1) ? 0 : selected
+      }
+    }
+
+    this.state = {
+      selected,
+      images: filteredImages
+    }
+  }
+
+  _getHeadImage () {
+    const { images, selected } = this.state
+    const { emptyImage } = this.props
+
+    const placeHolder = {
+      alt: 'Placeholder',
+      src: emptyImage
+    }
+
+    const src = (images.length === 0) ? emptyImage : images[selected].src
+    const alt = (images.length === 0) ? 'No text available' : images[selected].alt
```

Comment:
If the image exists and it has no alt, then the alt text (indexed) would actually be 'No text available'
Do you really want that.
You could also declare the shape of images to force alt texts as an accessibility requirement.